pymc3_models/models/NaiveBayes.py

```diff
@@ -0,0 +1,283 @@
+import functools as ft
+
+import numpy as np
+import pymc3 as pm
+import scipy.stats
+from sklearn.metrics import accuracy_score
+import theano
+
+from pymc3_models.exc import PyMC3ModelsError
+from pymc3_models.models import BayesianModel
+
+
+__all__ = ["GaussianNaiveBayes"]
+
+
+# Gaussian Naive Bayes
```

Comment:
I don't think you need this comment since it's the same as the classname.

Reply:
Fixed.